nirc_ehr/src/org/labkey/nirc_ehr/dataentry/section/NIRCCasesFormPanelSection.java

```diff
@@ -21,4 +32,18 @@ public JSONObject toJSON(DataEntryFormContext ctx, boolean includeFormElements)
         json.put("dataDependentCollapseHeader", true);
         return json;
     }
+
+    @Override
+    protected List<FieldKey> getFieldKeys(TableInfo ti)
+    {
+        List<FieldKey> keys = super.getFieldKeys(ti);
+
+        // only Vets and Folder Admins can see the enddate ('Close date') field to be able to close the case.
+        if (!isVet && (isVetTech || !isFolderAdmin))
```

Comment:
can someone be a vet tech and folder admin but 'vet'? I think this will remove the enddate for them too.